shopfloor_single_product_transfer/services/single_product_transfer.py

```diff
@@ -630,11 +618,15 @@ def _set_quantity__check_location(
                 move_line, message=message, asking_confirmation=confirmation or None
             )
 
-    def _write_destination_on_lines(self, lines, location):
-        stock = self._actions_for("stock")
-        stock.set_destination_and_unload_lines(lines, location)
+    def _write_destination_on_lines(self, lines, location, unload=False):
+        lines.picking_id.location_dest_id = location
```

Comment:
try by replacing by:
lines.location_dest_id = location
lines.package_level_id.location_dest_id = location

and drop below fixme

Reply:
@mmequignon ping